bin/zip-all.sh

```diff
@@ -49,12 +54,10 @@ getFileSizeMB () {
 
 print4ColsJustified "OS" "Uncompressed size (MB)" "Compressed size (MB)" "v$VERSION Reproducible build SHA-256"
 print4ColsJustified "------------------------" "------------------------" "------------------------" "------------------------------------------------------------------"
-print4ColsJustified "Mac"         "`getFileSizeMB $basedir/dist/mac`"       "`getFileSizeMB $basedir/dist-zip/i2p-zero-mac.v${VERSION}.zip`"       "\``getHash $basedir/dist-zip/i2p-zero-mac.v${VERSION}.zip`\`"
-print4ColsJustified "Windows"     "`getFileSizeMB $basedir/dist/win`"       "`getFileSizeMB $basedir/dist-zip/i2p-zero-win.v${VERSION}.zip`"       "\``getHash $basedir/dist-zip/i2p-zero-win.v${VERSION}.zip`\`"
-print4ColsJustified "Linux"       "`getFileSizeMB $basedir/dist/linux`"     "`getFileSizeMB $basedir/dist-zip/i2p-zero-linux.v${VERSION}.zip`"     "\``getHash $basedir/dist-zip/i2p-zero-linux.v${VERSION}.zip`\`"
-print4ColsJustified "Mac GUI"     "`getFileSizeMB $basedir/dist/mac-gui`"   "`getFileSizeMB $basedir/dist-zip/i2p-zero-mac-gui.v${VERSION}.zip`"   "\``getHash $basedir/dist-zip/i2p-zero-mac-gui.v${VERSION}.zip`\`"
-print4ColsJustified "Windows GUI" "`getFileSizeMB $basedir/dist/win-gui`"   "`getFileSizeMB $basedir/dist-zip/i2p-zero-win-gui.v${VERSION}.zip`"   "\``getHash $basedir/dist-zip/i2p-zero-win-gui.v${VERSION}.zip`\`"
-print4ColsJustified "Linux GUI"   "`getFileSizeMB $basedir/dist/linux-gui`" "`getFileSizeMB $basedir/dist-zip/i2p-zero-linux-gui.v${VERSION}.zip`" "\``getHash $basedir/dist-zip/i2p-zero-linux-gui.v${VERSION}.zip`\`"
+for i in ${TARGETS}; do
+  print4ColsJustified "${i}"         "`getFileSizeMB $basedir/dist/${i}`"       "`getFileSizeMB $basedir/dist-zip/i2p-zero-${i}.v${VERSION}.zip`"       "\``getHash $basedir/dist-zip/i2p-zero-${i}.v${VERSION}.zip`\`"
+  print4ColsJustified "${i} GUI"         "`getFileSizeMB $basedir/dist/${i}-gui`"       "`getFileSizeMB $basedir/dist-zip/i2p-zero-${i}-gui.v${VERSION}.zip`"       "\``getHash $basedir/dist-zip/i2p-zero-${i}-gui.v${VERSION}.zip`\`"
```

Comment:
I think this will fail if the targets includes e.g. "linux" but not also "linux-gui"?

Reply:
If one platform is included in $TARGETS, it will include both the cli and the gui, didn't do the effort to build only cli versus gui

Reply:
Ah, I see what you've done now, thanks